lib/internal/modules/cjs/loader.js

```diff
@@ -351,7 +352,7 @@ function resolveExports(nmPath, request, absoluteRequest) {
     const mappingKey = `.${expansion}`;
 
     if (typeof pkgExports === 'object' && pkgExports !== null) {
-      if (mappingKey in pkgExports) {
+      if (ObjectPrototype.hasOwnProperty(pkgExports, mappingKey)) {
```

Comment:
missing a .call

Reply:
@ljharb all things off `primordials` that are `Prototype`s uncurry `this`

Reply:
confusing, but makes sense